mypyc/irbuild/ll_builder.py

```diff
@@ -432,34 +433,34 @@ def literal_static_name(self, value: Union[int, float, complex, str, bytes]) ->
     def load_static_int(self, value: int) -> Value:
         """Loads a static integer Python 'int' object into a register."""
         if abs(value) > MAX_LITERAL_SHORT_INT:
-            static_symbol = self.literal_static_name(value)
-            return self.add(LoadStatic(int_rprimitive, static_symbol, ann=value))
+            identifier = STATIC_PREFIX + self.literal_static_name(value)
```

Comment:
Can you move the `STATIC_PREFIX + ...` calculation to `literal_static_name` so that it doesn't need to be duplicated several times?